source/org/zfin/infrastructure/repository/HibernateInfrastructureRepository.java

```diff
@@ -686,6 +687,21 @@ public ZdbFlag getUpdatesFlag() {
         return (ZdbFlag) criteria.uniqueResult();
     }
 
+    public FeatureFlag getFeatureFlag(String name) {
+        Session session = HibernateUtil.currentSession();
+        org.hibernate.query.Query query = session.createQuery("from FeatureFlag ff WHERE ff.name = :name");

```

Comment:
I don't think you need the alias 'ff'...

Reply:
Good point. Removed.